hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTargetManagement.java

```diff
@@ -182,10 +183,16 @@ public long count() {
 
         final JpaTarget updatedTarget = touch(target);
 
-        return Collections.unmodifiableList(md.stream()
+        final List<TargetMetadata> createdMetadata = Collections.unmodifiableList(md.stream()
                 .map(meta -> targetMetadataRepository
                         .save(new JpaTargetMetadata(meta.getKey(), meta.getValue(), updatedTarget)))
                 .collect(Collectors.toList()));
+
+        // target update event is set to ignore "lastModifiedAt" field so it is
```

Comment:
I would reformulate it as: TargetUpdatedEvent is not sent within the touch() method due to the "lastModifiedAt" field being ignored in JpaTarget